src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,134 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.utility.Storage;
+import duke.utility.TaskList;
+import duke.utility.Ui;
+
+/**
+ * Class to represents all command. This class also contains specific string for commands.
+ */
+public abstract class Command {
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_DELETE = "delete";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_TASK_AFTER = "taskafter";
+    private static final String COMMAND_TASK_BEFORE = "taskbefore";
+    private static final String COMMAND_FIND = "find";
+    private boolean isExit;
+
+    /**
+     * Constructs a Command object with isExit as false.
+     */
+    public Command() {
+        this.isExit = false;
+    }
+
+    /**
+     * Returns whether the Duke should stop running or not.
+     * @return Duke's current state to continue running.
+     */
```

Comment:
There can be an empty space between the description and parameter section, I noticed this in the other sections too.